x-pack/plugins/session_view/public/components/process_tree_node/index.tsx

```diff
@@ -168,6 +166,7 @@ export function ProcessTreeNode({
                     [exit_code: {exitCode}]
                   </small>
                 )}
+                {timeStampOn ? <span css={styles.timeStamp}>{timeStampsNormal}</span> : null}&nbsp;
```

Comment:
you can simplify it with`{timeStampOn && ...}`

Reply:
do we really need `&nbsp;` if we have `paddingRight` in `styles.timeStamp` already?